pkg/util/fastmapper.go

```diff
@@ -55,7 +55,7 @@ func (m *FastRESTMapper) Discover() error {
 			VersionedResources: make(map[string][]metav1.APIResource),
 		}
 		grs = append(grs, gr)
-		wg.Start(func() { m.DiscoverGroup(gr) })
+		wg.Start(func() { Panic(m.DiscoverGroup(gr)) })
```

Comment:
I don't think Panic is suitable here. Discover can fail due to disconnection.
Lets create a new func `util.LogError()` that will log if there was an error but continue.